docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -185,6 +202,23 @@ type ExpoAssetHeaderDictionary = {
 
 - `assetRequestHeaders`: MAY contain a dictionary of header (key, value) pairs to include with asset requests. Key and value MUST both be strings.
 
+### Directive Body
+
+Defined as JSON conforming to both the following `Directive` definition expressed in [TypeScript](https://www.typescriptlang.org/) and the detailed descriptions for each field:
+
+```typescript
+type Directive = {
+  directiveType: string;
+  directivePayload?: { [key: string]: any };
+  [key: string]: any;
```

Comment:
As I understand it, these extra fields are allowed for forwards-compatibility. The spec should say to allow these fields and to ignore any that aren't documented by the spec so that a client doesn't start relying on unspecced fields.

Reply:
These extra fields are for things like code signing context (scopeKey, projectId) and anything else that a client-server implementation may need. Maybe we should specify an `extra` field as well with the same style as `manifest.extra` (for `scopeKey`, `eas.projectId`). 

Also, more generally this spec says: 

> A conforming implementation of this protocol MAY provide additional functionality, but MUST NOT where explicitly disallowed or would otherwise result in non-conformance.

So I think I'll specify `extra` in the spec and remove the `[key: string]: any;`.

Reply:
I do think it's a good idea to have the spec tell clients to allow and ignore unknown fields. It will help with forward compatibility if old clients will gracefully ignore fields instead of crashing if a new server sends back unknown fields.